docs/modules/operation/pages/deep-dive/admin/webui/introduction.adoc

```diff
@@ -1,4 +1,63 @@
 
-= Administrative Web Interface
+= Admin UI Customization
 
-This section describes the main sections of the {page-component-title} web UI.
\ No newline at end of file
+This section describes how to customize the main components of the {page-component-title} web UI.
+
+== Dashboard
+
+A dashboard is an at-a-glance overview of your monitored network; it can be used as a starting point for more detailed diagnoses of issues.
+You can customize your dashboard using its predefined components.
+
+For more information, see xref:deep-dive/admin/webui/dashboard.adoc[].
+
+== Grafana Dashboard Box
+
+The Grafana Dashboard Box uses Grafana to generate visualizations.
+By default, it shows a set of dashboards related to {page-component-title} and your monitored environment.
+You can filter the displayed information and configure the Dashboard Box as desired.
+
+For more information, see xref:deep-dive/admin/webui/grafana-dashboard-box.adoc[].
```

Comment:
This "Grafana Dashboard Box" is separate from the "Configure Grafana Endpoints" item in the Admin UI.  The latter is documented in the Reports section, but the docs linked here reference things outside the Admin UI.